salt/netapi/rest_cherrypy/app.py

```diff
@@ -2326,7 +2326,6 @@ def signal_handler(signal, frame):
                         logger.error(
                                 "Error: Salt event has non UTF-8 data:\n{0}"
                                 .format(data))
-                time.sleep(0.1)
```

Comment:
To my first comment, I think it was needed pre-Tornado, which now does now [the same thing here](https://github.com/tornadoweb/tornado/blob/master/tornado/gen.py#L936). OK, _I think_ that it is safe to remove after all.